taipy/gui/gui.py

```diff
@@ -396,7 +396,28 @@ def __init__(
         if libraries is not None:
             for library in libraries:
                 Gui.add_library(library)
+    def set_on_invalid_data_callback(self, callback: t.Callable[[t.Any], t.Optional[t.Any]]):
```

Comment:
on_invalid_data should be handled the same way as on_action

- [definition](https://github.com/Avaiga/taipy/blob/3297952c0e64bd10358fde8e63a3adf6aef4e613/taipy/gui/gui.py#L255)
- [assignment](https://github.com/Avaiga/taipy/blob/3297952c0e64bd10358fde8e63a3adf6aef4e613/taipy/gui/gui.py#L2583)